template-processors/openshift-provision/files/processTemplates.yml

```diff
@@ -0,0 +1,42 @@
+---
+- name: Required variable check
+  hosts: localhost
+  gather_facts: false
+
+  pre_tasks:
+    - include_vars:
+        file: "{{ config_dump }}"
+    - set_fact:
+        openshift_resource_path: []
+        openshift_cluster_provision_pre_tasks: []
+        openshift_cluster_provision_post_tasks: []
+
+    - name: Generate openshift_resource_path
+      set_fact:
+        openshift_resource_path: "{{ openshift_resource_path }} + [ '/{{ item }}/resources' ]"
```

Comment:
why the `/` at the beginning of the paths? we want to make them start from filesystem root for some reason?